### DISCUSSION TITLE
v0.6.0

### DISCUSSION THREAD

**Discussion by erikerlandson** [erikerlandson/coulomb#363] [General]
The first official Scala 3 release for coulomb :tada: 

Essentially the same as `v0.6.0-RC1` except:
- scalafmt based code formatting and checking is now enabled
- I added a development section to the site documentation to help prospective contributors and also help me remember all the sbt commands

<hr /><em>This discussion was created from the release <a href='https://github.com/erikerlandson/coulomb/releases/tag/v0.6.0'>v0.6.0</a>.</em>

**Comment by cquiroz** [comment]
Congratulations on the final release, it is a core library in our projects and even better now in scala 3

**Reply by erikerlandson** [reply]
@cquiroz thanks to you and @armanbilge for all your help!  I'm looking forward to filling in some of the missing pieces in future releases.

**Reply by armanbilge** [reply]
Congratulations indeed!! The documentation is fantastic 😁
